tests/unit/kvmagent/test_mevoco_handlers.py

```
@@ -990,6 +1005,56 @@ def _bash_r(cmd: str) -> int:
 
 @pytest.mark.kvmagent
 class TestMevocoApplyUserdataInternals:
+    def test_validate_ebtables_name_rejects_shell_meta(self):
+        ebtables_mod = _load_real_ebtables_module()
+        with pytest.raises(Exception):
```

Ruff (0.15.14)

[warning] Do not assert blind exception: `Exception`

(B017)